internal/rest/k8s_utils.go

```diff
@@ -76,21 +76,6 @@ func (rest *RestOperations) SyncObjectStatuses() {
 						VirtualServiceUUID: vsCacheObj.Uuid,
 					})
 			}
-		} else if len(vsSvcMetadataObj.NamespaceServiceName) > 0 {
```

Comment:
@chauhanshubham : Sorry, I could have missed this during original PR. I am not getting purpose of removing LBUpdateOption block from here and adding it two places. 
 I think for each L4 service, vskey will be always different and will be part of vsKeys fetched.  so i think we should not treat it with insecure vs way.
 Also all changes doesn't seems to be specific to wcp(gw).